ttnn/cpp/ttnn/operations/data_movement/view/view.cpp

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "view.hpp"
+#include "ttnn/cpp/ttnn/operations/data_movement/reshape_view/reshape.hpp"
+
+namespace ttnn::operations::data_movement {
+
+ttnn::Tensor ViewOperation::invoke(const ttnn::Tensor& tensor, const ttnn::Shape& input_shape) {
+    auto layout = tensor.get_layout();
```

Comment:
Could we make the SimpleShape version be the one with the main code. Shape should go away at some point.

That also makes me wonder if there should be no Shape version and require users to use SimpleShape.

Reply:
I am using a bunch of utilities that require Shape and not SimpleShape. Now I know we can turn a SimpleShape into a Shape, can the same be done in the reverse direction so that I can use those utilities?

Reply:
You should be able to use the following of Shape's methods:
- padded_shape() and
- logical_shape()

If that's not sufficient, maybe explain the ops and their use cases and @ayerofieiev-tt and @TT-BrianLiu might be able to comment more.

Reply:
I think there's also the tensor's physical_shape() method.

Reply:
I have migrated over to SimpleShape